examples/ingress-recommendations/README.md

```diff
@@ -0,0 +1,175 @@
+# Ingress recommendations
+
+This example scenario showcases the BotKube recommendations feature around Ingress resources.
+
+## Prerequisites
+
+### Software
+
+Install the following applications:
+
+- [colima](https://github.com/abiosoft/colima)
+- [Helm](https://helm.sh/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Mkcert](https://github.com/FiloSottile/mkcert)
+
+### Set up local Kubernetes cluster
+
+Run the command:
+
+```bash
+colima start --runtime containerd --kubernetes --cpu 2 --memory 3 --disk 20 --profile botkube
+```
+
+### Install Ingress Nginx Controller
+
+Run the command:
+
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace --wait \
+  -f ./examples/ingress-recommendations/ingress-overrides.yaml
+```
+
+### Generate and use self-signed certificate
+
+1. Generate the certificate:
+
+  ```bash
+  mkcert "*.botkube.local" botkube.local localhost 127.0.0.1 ::1
+  ```
+
+1. Create Kubernetes secret:
+
+  ```bash
+  CERT_CRT=$(echo -n "$(cat ./_wildcard.botkube.local+4.pem)" | base64)
+  CERT_KEY=$(echo -n "$(cat ./_wildcard.botkube.local+4-key.pem)" | base64)
+  cat > /tmp/secret.yaml << ENDOFFILE
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: default-ssl-cert
+    namespace: ingress-nginx
+  type: kubernetes.io/tls
+  data:
+    tls.crt: ${CERT_CRT}
+    tls.key: ${CERT_KEY}
+  ENDOFFILE
+
+  kubectl apply -n ingress-nginx -f /tmp/secret.yaml
+  ```
+
+1. Add entry to `/etc/hosts`:
+
+  ```bash
+  readonly DOMAIN="botkube.local"
+  readonly BOTKUBE_HOSTS=("example")
+
+  LINE_TO_APPEND="127.0.0.1 $(printf "%s.${DOMAIN} " "${BOTKUBE_HOSTS[@]}")"
+  HOSTS_FILE="/etc/hosts"
+
+  grep -qF -- "$LINE_TO_APPEND" "${HOSTS_FILE}" || (echo "$LINE_TO_APPEND" | sudo tee -a "${HOSTS_FILE}" > /dev/null)
+  ```
+
+1. Navigate to https://example.botkube.local/ - you should see 404 error, but the connection should be secured.
```

Comment:
Suggestion: `Navigate to https://example.botkube.local/ in your browser and you should see 404 error page, but the connection should be secured.`